packages/ember-runtime/lib/system/array_proxy.js

```diff
@@ -33,6 +35,40 @@ var EMPTY = [];
 
 function K() { return this; }
 
+export function removeAt(array, start, len) {
+  if ('number' === typeof start) {
```

Comment:
the first argument is still required to be an ember array.

This isn't the equivalent of Ember.set(obj, key, val) where the obj can be a pojo.